src/fallbacks.jl

```diff
@@ -58,7 +58,7 @@ function rows(x::T) where {T}
     if columnaccess(T)
         cols = columns(x)
         return RowIterator(cols, rowcount(cols))
-    elseif Base.isiterable(T)
+    elseif TableTraits.isiterabletable(x) === true || Base.isiterable(T)
```

Comment:
I think we could further simplify this: `TableTraits.isiterabletable(x)` will return `true` if `Base.isiterable(T)` is `true`, so we don't need that second test. Also, I think we also want to take this branch if `TableTraits.isiterabletable(x)===missing`, right? So I think the test should probably read `elseif TableTraits.isiterabletable(x) !== false`.

But having said this, this PR is clearly fixing a bug already, so thanks for opening it!

Reply:
You're clearly the expert here, so feel free to open a PR to improve this.

Reply:
Will do once I'm done with my current travel stint. @quinnj could you tag a bug fix release that includes this PR soon? Tables v0.2.0 broke the entire file IO story in Queryverse, and this PR fixes that.